pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1095,7 +1095,7 @@ func (dm *DockerManager) pluginDisablesDockerNetworking() bool {
 
 // newDockerVersion returns a semantically versioned docker version value
```

Comment:
I don't see anyone is using this function at all! except docker_manager_test.go. If no objection from anyone, I would like to remove this function completely.